<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 5 - Terrain</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60000</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="cityDoc">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>205</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524344</width>
     <height>524275</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>&amp;CityGen</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="CityDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QWidget" name="filterDockEmptySpace" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QGroupBox" name="settingSelect">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>341</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QCheckBox" name="parkTog_2">
           <property name="text">
            <string>Toggle Park</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="treeTog_2">
           <property name="text">
            <string>ToggleTrees</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="grassTog">
           <property name="text">
            <string>Toggle Grass</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="seedLabel_2">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Seed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="seedSlider">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="seedEdit">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rotHLabel">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Rot Horizontal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="rotHSlider">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rotEditH">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rotVLabel">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Rot Vertical</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="rotVSlider">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rotEditV">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="spinTog">
           <property name="text">
            <string>Sp&amp;in</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="WalkTog">
           <property name="text">
            <string>Wal&amp;k</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parkTog_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>treeTog_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parkTog_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>grassTog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
